frontend/src/pages/articles/components/reactions/reactions.tsx

```diff
@@ -0,0 +1,38 @@
+import { Icon } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  comments: string;
+  views: string;
+  likes: string;
+  dislikes: string;
+};
+
+const Reactions: React.FC<Properties> = ({
+  comments,
+  views,
+  likes,
+  dislikes,
+}) => (
+  <ul className={styles.reactions}>
+    <li className={styles.reaction}>
+      <Icon iconName="comment" className={styles.reactionIcon} />
+      <span className={styles.reactionCount}>{comments}</span>
+    </li>
+    <li className={styles.reaction}>
+      <Icon iconName="view" className={styles.reactionIcon} />
+      <span className={styles.reactionCount}>{views}</span>
+    </li>
+    <li className={styles.reaction}>
+      <Icon iconName="like" className={styles.reactionIcon} />
+      <span className={styles.reactionCount}>{likes}</span>
+    </li>
+    <li className={styles.reaction}>
+      <Icon iconName="dislike" className={styles.reactionIcon} />
+      <span className={styles.reactionCount}>{dislikes}</span>
+    </li>
+  </ul>
```

Comment:
Should be in article-card component

Reply:
What about the tags, do I need to move them here too? I'm a bit confused because in the previous task I was told that it's better to move the tags to a separate component.

Reply:
Tags may be in separate component - as they're just displaying some sort of information (and will do it later on)